sockets/utils/getCurrentScriptSource.js

```diff
@@ -11,7 +11,8 @@ function getCurrentScriptSource() {
 
   // Fallback to getting all scripts running in the document.
   const scriptElements = document.scripts || [];
-  const currentScript = scriptElements[scriptElements.length - 1];
+  const nonNullScriptElements = [...scriptElements].filter((elm) => elm.getAttribute('src'));
```

Comment:
Unfortunately, we cannot use ES6 syntax here because we also need to support IE and older browsers.

```js
const scriptElementsWithSrc = Array.prototype.filter.call(scriptElements, function (elem) {
  return elem.getAttribute("src");
});
```

Reply:
Thanks for the quick reply! I can update that.